src/pages/api/auth/[...nextauth].ts

```diff
@@ -82,25 +85,25 @@ declare module 'next-auth' {
    * Session object available everywhere
    */
   export interface Session {
-    user: ServerUser
     accessToken: string
-    refreshToken: string
+    user: ServerUser | null
+    error?: string
```

Comment:
I can't find the usecase for this `error` prop as we assign only the other two

Reply:
Done in the last commit [34dde0f](https://github.com/podkrepi-bg/frontend/pull/779/commits/34dde0fd19678adbda16716ca6f8ee45a440fda4)